packages/employer-client/src/Applications/ApplicationCreate.tsx

```diff
@@ -1,16 +1,19 @@
+import React from "react"
```

Comment:
May be preferable to import useEffect and useState individually, to avoid unnecessary imports included in React, and for consistency with other places in this repo.